include/OpenColorIO/OpenColorIO.h

```diff
@@ -557,6 +606,35 @@ class OCIOEXPORT Config
     //!cpp:function::
     const char * getActiveViews() const;
 
+    //!cpp:function:: Get all displays in the config, ignoring the active_displays list.
+    int getNumDisplaysAll() const;
```

Comment:
Just my two cents, but I have sometimes found it less than ideal that `get/setActiveDisplays` and `get/setActiveViiews` get/set the raw comma-delimited strings instead of iterating over active display/view names like the other display/view methods. The current case of needing a `getDisplayAll` feels a bit awkward, as opposed to `getDisplay` returning all displays, and `getActiveDisplays` returning active displays. We could even add `serializeActiveViews` and `deserializeActiveViews` methods to accomodate the previous `get/setActive* `behavior. My suggestion would be a big API change so I'm not expecting it to be done, but raising it as a discussion point while this work is happening.

Reply:
as you explained, we wanted to preserve the behavior of getDisplay to only consider active displays. The all function is needed internally to be able to save all displays and we would not have exposed them to the API if the pimpl interface design would had allowed us to do so.

Reply:
Did you consider an overloaded `getDisplay` method with an argument to return all displays instead of creating additional interface? That was done with `getColorSpace` (reference space and visibility options).

Reply:
Actually I guess that won't work well with the index-based lookups... since index would then have variable behavior depending on display order. I concede.